ArduCopter/inertia.cpp

```diff
@@ -8,9 +8,10 @@ void Copter::read_inertia()
 
     // pull position from ahrs
     Location loc;
-    ahrs.get_position(loc);
-    current_loc.lat = loc.lat;
-    current_loc.lng = loc.lng;
+    if (ahrs.get_position(loc)) {
```

Comment:
this may mean that if EKF fails and copter crashes in long grass that the user can't find it. I think MissionPlanner only displays the GLOBAL_POSITION_INT pos. MAVProxy also displays GPS pos, but that won't help most users

Reply:
I agree this may not be the solution we want.  I think because of the odd behaviour of the AHRS::get_position() method we need to re-evaluate the EKF PR that changes the behaviour so that the EKF can return (0,0).  Instead the EKF should always keep returning it's best guess of what the last known position was.  A lack of confidence in that position is given to the vehicle code using the nav_flags.

Reply:
I can do that, and return last known position if GPs has failed,  but the boolean should  still be set to false.

Reply:
See https://github.com/ArduPilot/ardupilot/pull/15810/commits/70c759e28ebf201d7d3b6fa22e99e04e483aa619